src/components/HomeBanner/HomeBanner.jsx

```diff
@@ -0,0 +1,30 @@
+import { Fade } from "react-slideshow-image";
+import React from "react";
+import "./HomeBanner.css";
+import "react-slideshow-image/dist/styles.css";
+import HomeImg1 from "../../assets/HomeImg1.jpg";
+import HomeImg2 from "../../assets/HomeImg2.jpg";
+import HomeImg3 from "../../assets/HomeImg3.jpg";
+import HomeImg4 from "../../assets/HomeImg4.png";
+import HomeImg5 from "../../assets/HomeImg5.jpg";
```

Comment:
you can keep this in a separate file and call the array here

Reply:
will keep that in mind